internal/admin/admin.go

```diff
@@ -654,7 +654,7 @@ func (adm *Admin) updateLogStream(ctx context.Context, lsid types.LogStreamID, p
 
 	clusmeta, err := adm.mrmgr.ClusterMetadataView().ClusterMetadata(ctx)
 	if err != nil {
-		return nil, status.Errorf(codes.Unavailable, "update log stream: %s", err.Error())
+		return nil, status.Errorf(codes.Unavailable, "update log stream: get cluster metadata %s", err.Error())
```

Comment:
Because the method `ClusterMetadata` already returns an error with a message indicating the cause, it seems redundant.

Reply:
okay